src/client/js/oddone.local.ts

```diff
@@ -33,11 +33,11 @@ $(() => {
         });
 
     _mplib.MultiplayR.Host('theoddone',
-                           transport,
-                           document.getElementById('host-container'),
-                           (res) => {
-                               _mplib.messages.checkReturnMessage(res, 'roomId');
-                               setupJoin(res.message);
-                           });
+        transport,
```

Comment:
imo this makes for terrible reading. arguments should always be aligned with the indentation of the previous argument. if you prefer such a style, then the first argument should start on its own line